src/store/actions/movies-actions.js

```diff
@@ -0,0 +1,200 @@
+import {
+  GET_MOVIES,
+  NETWORK_PROVIDER_RESOLUTION,
+  DELETE_MOVIE,
+  UPDATE_MOVIE,
+  CREATE_MOVIE,
+} from './movies-action-types';
+
+const BASE_URL = 'http://localhost:4000';
+
+const generateFullEndpoint = (endpoint) => BASE_URL + endpoint;
+
+const headers = (method) => {
```

Comment:
it's better to use `getHeaders` instead of `headers`

Reply:
Fixed.